modules/module_utils/ansible_nailgun_cement.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
```

Comment:
We were not really sure which is the best way to place playbook and module_utils, since ansible needs it to be relative. We cannot really understand the sense behind it.
However we wanted a relative positioning of the module and the module_library "cement", so we placed the module_utils inside the modules folder and adapted our ansible.cfg.
We are also thinking about making a related PR at ansible, to enable a relative import of module_utils via the -M flag.